src/psyclone/tests/test_files/dynamo0p3/error_trans.py

```diff
@@ -14,5 +14,5 @@ def trans(psy):
     schedule = psy.invokes.get("invoke_0_testkern_type").schedule
     for child in schedule.children:
         if isinstance(child, Loop) and child.field_space != "w3":
-            cschedule, _ = ctrans.appy(child)
+            ctrans.appy(child)
```

Comment:
Since we're here, please could you disable the pylint no-member warning for this line.

Reply:
Done